<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1237</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>image</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="canvas">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="what">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>271</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Что нарисовать?</string>
    </property>
    <widget class="QRadioButton" name="ellips">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Эллипс</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="circle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Окужность</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="other">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>441</width>
      <height>741</height>
     </rect>
    </property>
    <property name="title">
     <string>Как нарисовать?</string>
    </property>
    <widget class="QLineEdit" name="A">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>320</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Параметры эллипса</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="radius">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>280</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>440</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Центр фигуры</string>
     </property>
    </widget>
    <widget class="QPushButton" name="line_color">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать цвет линии</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="brez">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Брезенхем</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="yc">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>230</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество фигур в спектре</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="line_color_win">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>510</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="middleDot">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Средней точки</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>570</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Радиус окружности</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bibl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Библиотечный</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="step">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>360</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>XC</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="B">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Шаг спектра</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="canon">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>201</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Каноническое уравнение</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="param">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Параметрические уравнение</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="amount">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>400</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>320</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="QPushButton" name="concentr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>610</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Спектр</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="bg_color_win">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>470</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>230</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>YC</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>381</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="xc">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bg_color">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Выбрать цвет фона</string>
     </property>
    </widget>
    <widget class="QPushButton" name="draw">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Построить одну фигуру</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1237</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
